boards/xtensa/intel_s1000_crb/Kconfig.defconfig

```diff
@@ -167,6 +145,38 @@ config GPIO_DW_0
 
 endif
 
+if FLASH
+
+config SPI
+	def_bool y
+
+config SPI_NOR
+	def_bool y
+
+config SPI_NOR_JEDEC_ID
```

Comment:
JEDEC ID, page size and sector size are discoverable (SFDP) and should be discovered instead of configured.

Reply:
Not applicable for this PR as specified earlier.